tools/cbmc/proofs/ARP/ARPGetCacheEntryByMac/Makefile.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ENTRY": "ARPGetCacheEntryByMac",
+  "CBMCFLAGS":
+  [
+      "--unwind 7"
```

Comment:
Since using `--nondet-static` seems to be the norm for FreeRTOS, please could you write a comment explaining why you don't use it here?

Reply:
There is no specific reason so I added the `--nondet-static` flag.